src/implementation/Client/GRPCClient/invoker.ts

```diff
@@ -30,20 +32,23 @@ export default class GRPCClientInvoker implements IClientInvoker {
   }
 
   // @todo: should return a specific typed Promise<TypeInvokerInvokeResponse> instead of Promise<nothing>
-  async invoke(appId: string, methodName: string, method: HttpMethod = HttpMethod.GET, data: object = {}): Promise<object> {
+  async invoke(appId: string, methodName: string, method: HttpMethod = HttpMethod.GET, data?: object,options:InvokerOptions={}): Promise<object> {
+    
+    const headers= options.headers!=undefined?options.headers:{}
```

Comment:
Can you change this? This is easier to read: `const headers = options.headers ?? {}`

Also on the `async invoke()` please use correct identation, so `async invoke(appId: string, methodName: string, method: HttpMethod = HttpMethod.GET, data?: object, options: InvokerOptions = {}):`